python/ray/tune/tune.py

```diff
@@ -1112,10 +1123,12 @@ def run_experiments(
         _ray_auto_init(entrypoint="tune.run_experiments(...)")
 
     if _remote:
-        if get_air_verbosity() is not None:
+        if get_air_verbosity(verbose) is not None:
             logger.warning(
-                "Ignoring AIR_VERBOSITY setting, "
-                "as it doesn't support ray client mode yet."
+                "[output] This will use the legacy output and progress reporter, "
+                "as Ray client is not supported by the new engine. "
+                "For more information, please see "
```

Comment:
for this one, and the identical Tuner output below, I think we should also give more context of what's happening and which setting causes this.

Reply:
This is explained in detail in the linked docs page, is that enough?

Reply:
hmm, I know this is just a warning, but it's still useful to make it actionable. If I run code and something fails, I want immediate feedback and not search the docs.

To be clear, I just want you to print the variable name and its state that causes this warning.

Reply:
I've changed this to an info log. There is no specific setting the user can change, it's just an information (so info actually makes sense)